namespace AsyncAwait.SyncContext._08_Void.Exception
{
    internal class VoidException : System.Exception
    {
        public VoidException(string message) : base(message)
        {
        }
    }
}
